submissions/v4voloshyn/html-css-popup/styles.css

```diff
@@ -0,0 +1,344 @@
+@import url('https://fonts.googleapis.com/css2?family=Figtree:wght@400;500;700&display=swap');
+
+* {
+	box-sizing: border-box;
+	margin: 0;
+
+}
+
+body {
+	width: 100%;
+	height: 100vh;
+	font-family: 'Figtree', sans-serif;
+}
+
+ul {
+	text-decoration: none;
+	list-style-type: none;
+	padding: 0;
+	margin: 0;
+}
+
+a {
+	text-decoration: none;
+	color: inherit;
+}
+
+.visually-hidden {
+	appearance: none;
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	opacity: 0;
+	pointer-events: none;
+	visibility: hidden;
+}
+
+.logo {
+	cursor: pointer;
+	padding: 10px;
+	border-radius: 15px;
+	border: 2px solid transparent;
+}
+
+.logo:hover,
+.logo:focus {
+	border: 2px solid black;
+	border-radius: 15px;
+}
+
+.header {
+	margin-top: 20px;
+	padding: 0 20px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	height: 40px;
+	width: 100%;
+}
+
+.nav-list {
+	position: relative;
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	flex: 0 1 500px;
+	gap: 1.5rem;
+}
+
+.nav-item {
+	height: 100%;
+	display: flex;
+	align-items: center;
+	border-radius: 5px;
+	transition: all ease-in-out .3s;
+}
+
+.nav-link {
+	font-size: 18px;
+	text-transform: capitalize;
+	border: 2px solid transparent;
+	padding: 5px;
+	border-radius: 15px;
+}
+
+.nav-link:hover,
+.nav-link:focus {
+	outline: none;
+	border: 2px solid black;
+}
+
+.user-img {
+	width: 40px;
+	height: 40px;
+	display: block;
+	border-radius: 50%;
+}
+
+.popup-label {
+	cursor: pointer;
+	display: flex;
+	border-radius: 15px;
+	width: 40px;
+	height: 40px;
+	align-items: center;
+	transition: border ease-in-out .3s;
+}
+
+.popup-list {
```

Comment:
one class and so many properties 😞  Please try to optimize your code, it is not easy to review. You can refactor and simplify it, some of the properties are redundant